### ISSUE TITLE
feat(ACCESSIBILITE): 

### ISSUE THREAD

**Issue by celinelr** [SocialGouv/domifa#2216]
# P06	Domiciliés	https://domifa.fabrique.social.gouv.fr/manage

Au moins un lien n'a pas d'intitulé.
Par exemple :
- Les liens dans les cellules vides.
Corrections :
- Supprimer ces liens.

**Comment by ErceauFloriane** [comment]
Pas de liens dans les cellules vides

**Comment by ErceauFloriane** [comment]
Environnement de test : https://domifa-feat-rgaa-test.dev.fabrique.social.gouv.fr/
@celinelr

**Comment by celinelr** [comment]
Je te laisse me corriger si je me trompe, mais dans le cas ou cette cellule est vite j'ai bien un lien. L'audit ferait référence à cela ? 

![Capture d’écran 2022-05-06 à 17 59 41](https://user-images.githubusercontent.com/86293911/167169416-4307355b-3fc5-42f1-92c4-4d4fd4fcb025.png)

**Comment by ErceauFloriane** [comment]
Effectivement, je n'ai pas osé les enlever, j'ai supprimer les liens c'est ok 

![Capture d’écran 2022-05-10 à 11 18 52](https://user-images.githubusercontent.com/67644911/167594873-d65cf639-651e-42d2-87c8-5bcddb56a986.png)

**Comment by celinelr** [comment]
Actuellement si l'usager n'a pas de numéro de téléphone, la case et vide, il faut supprimer le numéro de téléphone

**Comment by celinelr** [comment]
Merci, je valide

**Comment by celinelr** [comment]
Je valide sur la preprod
